### PR TITLE
Add Wayne file

### PR DESCRIPTION
**Pull request by fjmurray**
Look through proposed changes and, if interested, please submit any feedback. 🇺🇸

### CODE REVIEW

File: fjmurray-Wayne.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Wayne, PA
+
+### Restaurants
+
+### Local Attractions
+
+- Philadelphia
```
Comment (githubteacher) [inline fjmurray-Wayne.md]: @fjmurray I've only been to the airport ✈️  -- but I hear that Philly has a lot of history. Do you have any suggestions?

Comment (fjmurray) [inline fjmurray-Wayne.md]: Independence Hall, and Olde City.

**Comment by fjmurray** [comment]
Keep getting "Merge attempt failed."

**Comment by fjmurray** [comment]
👎

**Comment by fjmurray** [comment]
👎

**Comment by fjmurray** [comment]
👍

**Pull request merged**